sdk/azcore/policy_retry_test.go

```diff
@@ -502,7 +501,44 @@ func TestRetryPolicySuccessWithPerTryTimeout(t *testing.T) {
 	defer close()
 	srv.AppendResponse(mock.WithSlowResponse(5 * time.Second))
 	srv.AppendResponse(mock.WithStatusCode(http.StatusOK))
-	pl := NewPipeline(srv, NewRetryPolicy(testRetryOptions()))
+	opt := testRetryOptions()
```

Comment:
This is confusing the diff for some reason.  This is the only part that changed in this test.